assets/css/_ladder.scss

```diff
@@ -1,4 +1,6 @@
 .m-ladder {
+  grid-column: span 1;
```

Comment:
I think `span 1` is the default. Have you tried without it or `grid-column: auto`?